visionos/ReactTestApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,565 @@
+		197827FA27710D3400AEC655 /* Manifest+Decoder.swift in Sources */ = {isa = PBXBuildFile; fileRef = 197827F927710D3400AEC655 /* Manifest+Decoder.swift */; };
+		1988284524105BEC005057FF /* UIViewController+ReactTestApp.m in Sources */ = {isa = PBXBuildFile; fileRef = 1988284424105BEC005057FF /* UIViewController+ReactTestApp.m */; };
+		19A624A4258C95F000032776 /* Session.swift in Sources */ = {isa = PBXBuildFile; fileRef = 19A624A3258C95F000032776 /* Session.swift */; };
+		19ECD0D6232ED425003D8557 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 19ECD0D5232ED425003D8557 /* AppDelegate.swift */; };
+		19ECD0D8232ED425003D8557 /* SceneDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 19ECD0D7232ED425003D8557 /* SceneDelegate.swift */; };
+		19ECD0DA232ED425003D8557 /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 19ECD0D9232ED425003D8557 /* ContentView.swift */; };
+		19ECD0DC232ED427003D8557 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 19ECD0DB232ED427003D8557 /* Assets.xcassets */; };
+		19ECD0ED232ED428003D8557 /* ReactTestAppTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 19ECD0EC232ED428003D8557 /* ReactTestAppTests.swift */; };
+		19ECD0F8232ED428003D8557 /* ReactTestAppUITests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 19ECD0F7232ED428003D8557 /* ReactTestAppUITests.swift */; };
+/* End PBXBuildFile section */
+
+/* Begin PBXContainerItemProxy section */
+		19ECD0E9232ED427003D8557 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = 19ECD0CA232ED425003D8557 /* Project object */;
+			proxyType = 1;
+			remoteGlobalIDString = 19ECD0D1232ED425003D8557;
+			remoteInfo = ReactTestApp;
+		};
+		19ECD0F4232ED428003D8557 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = 19ECD0CA232ED425003D8557 /* Project object */;
+			proxyType = 1;
+			remoteGlobalIDString = 19ECD0D1232ED425003D8557;
+			remoteInfo = ReactTestApp;
+		};
+/* End PBXContainerItemProxy section */
+
+/* Begin PBXFileReference section */
+		192DD200240FCAF5004E9CEB /* Manifest.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = Manifest.swift; path = ../ReactTestAppShared/Manifest.swift; sourceTree = "<group>"; };
+		192F052624AD3CC500A48456 /* ReactTestApp.release.xcconfig */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xcconfig; path = ReactTestApp.release.xcconfig; sourceTree = "<group>"; };
+		192F052724AD3CC500A48456 /* ReactTestApp.common.xcconfig */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xcconfig; path = ReactTestApp.common.xcconfig; sourceTree = "<group>"; };
+		192F052824AD3CC500A48456 /* ReactTestApp.debug.xcconfig */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xcconfig; path = ReactTestApp.debug.xcconfig; sourceTree = "<group>"; };
```

Comment:
Are the `.xcconfig` files for iOS close enough that we can reuse them?
```suggestion
		192F052624AD3CC500A48456 /* ReactTestApp.release.xcconfig */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xcconfig; path = ../ReactTestAppShared/ReactTestApp.release.xcconfig; sourceTree = "<group>"; };
		192F052724AD3CC500A48456 /* ReactTestApp.common.xcconfig */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xcconfig; path = ../ReactTestAppShared/ReactTestApp.common.xcconfig; sourceTree = "<group>"; };
		192F052824AD3CC500A48456 /* ReactTestApp.debug.xcconfig */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xcconfig; path = ../ReactTestAppShared/ReactTestApp.debug.xcconfig; sourceTree = "<group>"; };
```

Reply:
The only difference is they set iOS deployment target instead of xros deployment target

Reply:
Aha, maybe make them inherit the iOS ones and override that one prop?

Reply:
Do we want to do the same for macOS? The differences are minor
![image](https://github.com/microsoft/react-native-test-app/assets/6722175/573fe623-0e92-403a-b454-39424ab8d611)

Reply:
I made an attempt: https://github.com/microsoft/react-native-test-app/pull/1896
Happy to mess with naming now that there's an extra platform suffix.